src/pages/search-page/SearchPage.tsx

```diff
@@ -427,35 +427,35 @@ const SearchPage = () => {
         }}
         gap={2}
       >
-        {/* Ignore the FULL_LIST view for now, wait for future design */}
-        {selectedLayout !== SearchResultLayoutEnum.FULL_MAP && (
-          <Box>
-            <ResultSection
-              onClickCard={onClickCard}
+        <Box>
```

Comment:
I prefer follow the same patter then the ResultSection have an attribute like below showFullMap, where you can showFullList

Reply:
I modified to pass showFullList and showFullMap